groupie/src/main/java/com/genius/groupie/Item.java

```diff
@@ -93,13 +78,11 @@ public void bind(T viewBinding, int position, List<Object> payloads) {
         return this == item ? 0 : -1;
     }
 
-    public boolean isClickable() {
```

Comment:
Ok, what do you think the default should be ?  
I found it very perturbing to have items clickable by default, but maybe it is just me